packages/gluestick-cli/index.js

```diff
@@ -47,6 +47,31 @@ commander
   });
 
 commander
+  .command('watch')
+  .description('applying changes in gluestick folder')
+  .action(() => {
+    let srcPath = require(path.join(process.cwd(), 'package.json')).dependencies.gluestick;
+    srcPath = srcPath.replace('file:', '');
+    srcPath = path.join(process.cwd(), srcPath);
```

Comment:
All these 3 lines can be e.g. extracted to function. Anyway, please try to rewrite it using `const srcPath` and avoiding overriding references.